File: my-folder/0001-two-sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        true = True
        while true:
            for number in range(len(nums) - 1):
                for add in range (number + 1, len(nums)):
                    outcome = nums[number] + nums[add]
                    if outcome == target:
                        return [number, add]
    
                     
                    
                   
               
                        
            
            




        

File: my-folder/0001-two-sum/solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        
      std::unordered_map<int, int> seen;

      for (int i=0; i < nums.size(); i++) {
        int looking = target - nums[i];
        if (seen.find(looking) != seen.end() ) {
            return {i, seen[looking]};

        }
        seen[nums[i]] = i;
      }      
      return {};
    }
};

